codemeta.json

```diff
@@ -0,0 +1,597 @@
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Point 8 GmbH, Rheinlanddamm 201, 44139 Dortmund, Germany"
+            },
+            "familyName": "Br\u00fcgge",
+            "givenName": "Kai"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Institute de Recherche en Astrophysique et Planetologie, IRAP,"
+            },
+            "familyName": "Tibaldo",
+            "givenName": "Luigi"
+        },
+        {
+            "@type": "Person",
+            "familyName": "Gogia",
+            "givenName": "Arpit"
+        },
+        {
+            "@type": "Person",
+            "familyName": "Minaya",
+            "givenName": "Ignacio"
+        },
+        {
+            "@type": "Person",
+            "familyName": "Spir-Jacob",
+            "givenName": "Marion"
+        },
+        {
+            "@type": "Person",
+            "familyName": "Gallant",
+            "givenName": "Yves"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Department of Physics, University of the Free State, PO Box339, Bloemfontein 9300, South Africa"
+            },
+            "familyName": "W. Chen",
+            "givenName": "Andrew"
+        },
+        {
+            "@id": "https://orcid.org/0000-0001-6320-1801",
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "CTA Observatory gGmbH, Via Piero Gobetti 93/3 40129 Bologna, Italy"
+            },
+            "familyName": "Zanin",
+            "givenName": "Roberta"
+        },
+        {
+            "@type": "Person",
+            "familyName": "Lenain",
+            "givenName": "Jean-Philippe"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Space Telescope Science Institute, STScI, 3700 San Martin Drive, 21218 Baltimore MD, US"
+            },
+            "familyName": "Bradley",
+            "givenName": "Larry"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Friedrich-Alexander-Universit\u00e4t Erlangen-N\u00fcrnberg, Erlangen Centre for Astroparticle Physics, Erwin-Rommel-Str. 1 91058 Erlangen, Germany"
+            },
+            "familyName": "Nakashima",
+            "givenName": "Kaori"
+        },
+        {
+            "@type": "Person",
+            "affiliation": {
+                "@type": "Organization",
+                "name": "Astroparticle and Cosmology Laboratory CNRS, Universit\u00e9 de Paris, 10 Rue Alice Domon et L\u00e9onie Duquet, 75013 Paris, France"
```

Comment:
the name of her affiliation is:
"Astroparticle and Cosmology Laboratory CNRS, Universit\u00e9 de Paris Cit\u00e9, 10 Rue Alice Domon et L\u00e9onie Duquet, 75013 Paris, France"